src/validation.h

```diff
@@ -247,9 +247,10 @@ bool ProcessNewBlock(const CChainParams& chainparams, const std::shared_ptr<cons
  * @param[in]  block The block headers themselves
  * @param[out] state This may be set to an Error state if any error occurred processing them
  * @param[in]  chainparams The params for the chain we want to connect to
+ * @param[out] store the invalid block here if there is one. 
```

Comment:
The new parameter was added to the end of the method, but this line makes it seem like it is located where the 'ppindex' is. I suggest swapping line 250 with line 251.